video/out/wayland_common.c

```diff
@@ -704,6 +705,10 @@ static void data_offer_handle_offer(void *data, struct wl_data_offer *offer,
         talloc_replace(wl, o->mime_type, mime_type);
         MP_VERBOSE(wl, "Given data offer with mime type %s\n", o->mime_type);
     }
+    if (o->offer && !o->offered_plain_text) {
+        if (strcmp(mime_type, "text/plain;charset=utf-8") == 0)
+            o->offered_plain_text = true;
+    }
```

Comment:
Is there a reason to not just always directly set `o->offered_plain_text` based on `strcmp` as long as `o->offer` exists?

Reply:
I could assign `o->offered_plain_text` to `!strcmp` instead of explicitly assigning it to `true` to remove one line, but once it's set to `true` we have to stop doing comparisons since it may get set back to `false` if the next offer events are for `UTF8_STRING`/`TEXT`/`chromium/x-source-url`/`text/html` etc.

Reply:
Now directly assigned to !strcmp